test/cmd/apply.sh

```diff
@@ -268,6 +268,29 @@ __EOF__
   # cleanup
   kubectl delete ns nsb
 
+  ## kubectl apply --prune --all --namespace NS can only prune resources in the namespace
+  # Pre-Condition: namespace nsb exists; no POD exists
+  kubectl create ns nsb
+  kube::test::get_object_assert pods "{{range.items}}{{${id_field:?}}}:{{end}}" ''
+  # apply a into namespace nsb
+  kubectl apply --namespace nsb -f hack/testdata/prune/a.yaml "${kube_flags[@]:?}"
+  kube::test::get_object_assert 'pods a -n nsb' "{{${id_field:?}}}" 'a'
+  # apply b with namespace
+  kubectl apply --namespace nsb -f hack/testdata/prune/b.yaml "${kube_flags[@]:?}"
```

Comment:
Currently, we are applying both of a.yaml and b.yaml  in `nsb` namespace. I'd expect applying some non-namespaced resource and see it is not pruned?

Reply:
You're right.